grid-client/state/state.go

```diff
@@ -322,6 +330,24 @@ func (st *State) LoadNetworkFromGrid(name string) (znet workloads.ZNet, err erro
 	znet.NodesIPRange = nodesIPRange
 	znet.Keys = keys
 	znet.WGPort = wgPort
+	if metdata.UserAcessIP != "" {
```

Comment:
this also should be moved to `NewNetworkFromWorkload`

Reply:
same as above